### ISSUE TITLE
The quickstart tutorial project does not work with Hilla 1.0.5

### ISSUE THREAD

**Issue by Artur-** [vaadin/hilla#385]
```
npx @vaadin/cli init --preset hilla-quickstart-tutorial hilla-grocery-app
cd hilla-grocery-app
mvn
```

Shows a white page in the browser and
```
Uncaught SyntaxError: The requested module '/VAADIN/@fs/<path-to-project>/hilla-grocery-app/node_modules/lit/index.js?v=34983637' does not provide an export named 'LitElement' (at lit-mobx.ts:14:1)
```

**Comment by tarekoraby** [comment]
I cannot reproduce this.

**Comment by platosha** [comment]
The issue is related with `@vaadin/bundles`, fixed in 23.0.6.
